docs/develop/javascript-client.md

```diff
@@ -157,6 +157,69 @@ Beside using the `initOnPjaxLoad` flag and `unload` function mentioned in the [M
 
 > Note: Since javascript files in Yii are only loaded and executed once per full page load, some modules may have to be reinitialized after a pjax request. As mentioned in the [Module Guide](javascript-index.md) the modules `init` function is called in case of a pjax request only if the modules `initOnPjaxLoad` setting is set to `true`.
 
+#### Running your own Pjax calls
+
+> Note: the current implementation of the `\humhub\widgets\Pjax` widget is intended for use in the core only. Additionally, it limits the functionality of Yii2's `\yii\widgets\Pjax`.
+
+The following example is meant as an indication of how you may still use Pjax to update the content of an inline-element.
+It is, however, are not officially supported.
```

Comment:
You can drop the "are" from this and just use "It is, however, not officially supported."